src/IconButton/IconButton.spec.js

```diff
@@ -72,4 +77,18 @@ describe('<IconButton />', () => {
       assert.include(wrapper.props().style, hoveredStyle);
     });
   });
+  describe('prop: disableTouchRipple', () => {
+    it('does not disable the ripple effect by default', () => {
+      const wrapper = mountWithContext(
+        <IconButton />
+      );
+      assert.strictEqual(wrapper.find(TouchRipple).length, 1, 'should contain a TouchRipple descendent');
+    });
+    it('disables the ripple effect when disableTouchRipple={true}', () => {
```

Comment:
I think that it would be better to use shorter description. We can always read the tests to have more details. What about `disables the ripple effect` here. The comment applies to the other description.